src/js/utils/MetronomeUtil.js

```diff
@@ -91,5 +91,36 @@ module.exports = {
     });
 
     return rootTree;
+  },
+
+  parseJob(job) {
+    let {history}= job;
+
+    if (history == null) {
+
+      return job;
+    }
+
+    let {failedFinishedRuns = [], successfulFinishedRuns = []} = history;
+
+    failedFinishedRuns = failedFinishedRuns.map(function (jobRun) {
+
```

Comment:
Sorry, don't wanted to be harsh.
But reviewing is also work. 
And I kindly felt ignored here, which is also not so nice.
This 'extra line' thingy was already discussed in the other PR.